index.html

```diff
@@ -6,24 +6,26 @@
   </head>
   <body>
     <header>
-      <h2>Steve Bachtel</h2>
+      <h2 id = "title">Steve Bachtel</h2>
     </header>
     <nav>
       <ul>
-        <li><a href="http://www.google.com">Google</a></li>
-        <li><a href="http://www.reddit.com">Reddit</a></li>
-        <li><a href="http://www.craigslist.org">Craigslist</a></li>
-        <li><a href="http://www.wsdot.com/ferries/schedule/ScheduleDetailByRoute.aspx?route=sea-bi">Bainbridge Ferry </a></li>
-        <li><a href="http://www.fark.com">Fark</a></li>
+        <li class = "navBar"><a href="http://www.google.com">Google</a></li>
+        <li class = "navBar"><a href="http://www.reddit.com">Reddit</a></li>
+        <li class = "navBar"><a href="http://www.craigslist.org">Craigslist</a></li>
+        <li class = "navBar"><a href="http://www.wsdot.com/ferries/schedule/ScheduleDetailByRoute.aspx?route=sea-bi">Bainbridge Ferry </a></li>
+        <li class = "navBar"><a href="http://www.fark.com">Fark</a></li>
       </ul>
```

Comment:
Close the spaces for your id and class attributes. Should id=", class=". For html, there shouldn't be any spaces between attributes and = signs.